metricq/synchronous_source.py

```diff
@@ -28,8 +28,9 @@
 # NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 import asyncio
+from collections.abc import Mapping
 from threading import Event, Lock, Thread
-from typing import Any, Dict
+from typing import Any
 
 from .logging import get_logger
 from .source import MetadataDict, Source
```

Comment:
Now we are importing `MetadataDict`, either replace it with `JsonDict`, or move that to timeseries as well.

Reply:
Why?

Reply:
So they are together.